### PR TITLE
Added email template and finished /unsubscribe API

### PR DESCRIPTION
**Pull request by RushabhDalwadi**

### CODE REVIEW

File: backend/AsterixUtils.py [hunk 1]
```diff
@@ -1,11 +1,17 @@
 import requests
 import json
 
-ASTERIX_API_URL = "http://localhost:19002/query/service"
+ASTERIX_HOSTNAME = "cacofonix-1.ics.uci.edu"
```
Comment (chriswongtv) [inline backend/AsterixUtils.py]: Move the DB hostnames to a .env file

File: backend/EmailUtils.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    finalHtmlStr = templateStr.format(tableRowStr)
+    print(finalHtmlStr)
+    return finalHtmlStr
+
+generateEmailHtmlStr()
+
+'''
+Send email on CentOS using the "mail" command.
+Example:
+echo "<html> Hello </html>" | mail -s "$(echo -e "Release Status [Green]\nContent-Type: text/html")" rdalwadi@uci.edu
```
Comment (chriswongtv) [inline backend/EmailUtils.py:22]: Move the email address to the .env file too, so everyone can test it using their email locally.

**Review by chriswongtv — CHANGES_REQUESTED** [review]

File: backend/EmailUtils.py (hunk 2 above)
Comment (RushabhDalwadi) [reply backend/EmailUtils.py:22]: This is just the sample command.  The email with be based on the one provided when calling the /subscribe api.

Comment (chriswongtv) [reply backend/EmailUtils.py:22]: Ah okay, got it!

**Review by chriswongtv — APPROVED** [review]

**Pull request merged**